pages/glossary/change-in-appearance.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Change in appearance
+key: change-in-appearance
+---
+
+A change in appearance for an element means that when comparing the element before and after the possible change has happened, the rendered pixels have changed within an area that includes the element plus a 50 pixel margin to each side.
```

Comment:
You should probably consider timing for this as well. Is this change in appearance is supposed to happen in the first frame paint after the change? If not, should there be a delay?